Source/Dafny/DafnyAst.cs

```diff
@@ -10392,7 +10400,6 @@ public enum CoCallResolution {
 
     public Function Function;  // filled in by resolution
 
-    [Captured]
     public FunctionCallExpr(IToken tok, string fn, Expression receiver, IToken openParen, [Captured] List<ActualBinding> args, Label/*?*/ atLabel = null)
```

Comment:
This overload could just call the one below with `new ActualBindings(args)`, couldn't it?

Reply:
Good suggestion. Done.